core/domain/learner_playlist_services_test.py

```diff
@@ -100,162 +101,148 @@ def _get_all_learner_playlist_collection_ids(self, user_id):
             learner_playlist_model.collection_ids if
             learner_playlist_model else [])
 
-    def test_mark_exploration_to_be_played_later(self):
+    def test_adding_subscribed_exp_to_playlist_does_nothing_without_pos(self):
```

Comment:
Why is the "without pos" bit important? I don't see a corresponding "with pos" test anywhere...

Reply:
The corresponding "with pos" test is actually the next thing that you commented on: test_adding_existing_exp_to_playlist_does_not_add_again_with_pos. 
Without a position, if the exploration is already in the playlist, nothing should happen. 
With a position, if the exploration is already in the playlist, the exploration should be moved to that position in the playlist. 
Also, I need both of these test functions in order to be able to reach 100% coverage (test both of those sections of the code).

Reply:
Hang on, I'm a little confused. You're talking about "existing exp" here but this test is about "subscribed exp" -- which is it?

Reply:
It should be subscribed exp.